distribution/lib/Standard/Table/0.0.0-dev/src/Main.enso

```diff
@@ -18,6 +18,8 @@ export project.Data_Formatter.Data_Formatter
 export project.Delimited.Delimited_Format.Delimited_Format
 export project.Delimited.Quote_Style.Quote_Style
 
+export project.Generate_Rows_Range.Generate_Rows_Range
+
```

Comment:
Don't think we need to export it - we will only use as an argument within the function.

Reply:
It just makes the widget work for DB_Table. Which I think they should...